s30576-pj-NatanBrzozowski/zad_uprawnienia_dostepu.txt

```diff
@@ -0,0 +1,187 @@
+	Polecenie cd - sluzy do powrotu do poprzedniego katalogu. Bez wzgledu 
+	na obecne uprawnienia, jesli wczesniej bylismy w innym katalogu, mozemy 
+	wrocic do niego za pomocą polecenia cd -. To polecenie nie wymaga specjalnych 
+	uprawnien dostepu do katalogu.
+
+--------------------------------------------------------------------------------
+
+|=============================|
+|         PYTANIE 3:          |
+|=============================|
+W zespołach 2- lub 3-osobowych (w opisie zadania na githubie proszę umieścić 
+login osób z zespołu) wyprobowac mozliwosc komunikacji przez wspoldzielony plik: 
+na jednym z kont w zespole utworzyc pusty plik i przydzielic odpowiednie 
+uprawnienia dostepu (do pliku i do katalogu domowego). Wpisywac i odczytywac 
+komunikaty przy uzyciu polecen:
+echo tresc_komunikatu > plik
+cat plik
+Sprawdzic, jaki skutek powoduje zamiana operatora > na operator >> w poleceniu 
+echo. Uruchom takze drugi terminal i wykonaj w nim komende tail -f plik i powtorz 
+powyzsze cwiczenie w pierwszym terminalu (komunikacja za pomocą pliku).
+
+|-----------------------------|
+|      ODPOWIEDZ/KOMENTARZ:   |
+|-----------------------------|
+touch wspoldzielony_plik.txt 		--> tworze plik
+chmod 666 wspoldzielony_plik.txt  	--> nadaje uprawnienia dla wszystkich do czytania i zapisywania
+
+echo "Czesc!" > wspoldzielony_plik.txt
+drugi uzytkownik robi: cat wspoldzielony_plik.txt i wyswietla mu sie Czesc!
+
+zmieniajac operator > na >> wiadmosc zostanie dopisana np. 
+echo "Nowa wiadomosc" >> wspoldzielony_plik.txt drugi uzytkownik po wykonaniu 
+cat wspoldzielony_plik.txt zobaczy obie wiadomosci Czesc i Nowa wiadomosc
+
+przy pomocy tail -f mozemy obserwowac zmiany w pliku w czasie rzeczywistym
+w jednym terminalu odpalamy tail -f wspoldzielony_plik.txt
+a w drugim komunikujemy sie jak w czesniej za pomoca echo "Kolejna wiadomosc" >> wspoldzielony_plik.txt
+i zmiany beda widoczne 
+
+--------------------------------------------------------------------------------
+
+|=============================|
+|         PYTANIE 4:          |
+|=============================|
+Znalezc w swoim katalogu domowym podkatalog public_html (jesli go nie ma, to 
+utworzyc; musi sie on nazywac DOKLADNIE tak jak podano, pisane malymi literami 
+z podkresleniem zamiast spacji pomiedzy slowami). Umiescic w nim plik o nazwie 
+strona.html o nastepujacej zawartosci:
+<HTML>
+<BODY>
+<H1>To jest moja strona domowa</H1>
+</BODY>
+</HTML>
+Sprawdzic, jakie sa minimalne uprawnienia dostepu, ktore trzeba przydzielić do:
+katalogu domowego;
+katalogu public_html;
+pliku strona.html, aby zawartosc pliku była widoczna w przegladarce internetowej 
+pod adresem http://szuflandia.pjwstk.edu.pl/~twój_login/strona.html
+
+|-----------------------------|
+|      ODPOWIEDZ/KOMENTARZ:   |
+|-----------------------------|
+chmod 711 ~  						--> minimalne uprawnienia do katalogu domowego, uzytkownik ma pelne uprawnienia, ale inni tylko dostep do katalogu
```

Comment:
Minimalne to dosłownie minimalne. Użytkownik/właściciel nie potrzebuje mieć dostępu. Grupa także.